app/views/payouts/list.njk

```diff
@@ -54,8 +54,8 @@ Payments to your bank account - GOV.UK Pay
 {% set transactionsDownloadPath = routes.allServiceTransactions.download if filterLiveAccounts else routes.formattedPathFor(routes.allServiceTransactions.downloadStatusFilter, 'test') %}
 <div class="govuk-grid-column-two-thirds">
   {% for key, group in payoutSearchResult.groups %}
-    {% set dateString = group.date.format('DD MMMM') %}
-    <h2 class="govuk-heading-m">{{ dateString }}</h2>
+    {% set dateString = group.date.format('D MMMM YYYY') %}
```

Comment:
I believe this is a Moment object and according to https://momentjs.com/docs/#/displaying/ `YYYY` means the year in Moment, unlike [in Java where it means the week year](https://technology.blog.gov.uk/2021/03/24/how-we-fixed-a-strange-week-year-bug/).

Changing `DD` to`D` (no leading zero) is line with the GOV.UK style guide for dates: https://www.gov.uk/guidance/style-guide/a-to-z#dates